resources/assets/coffee/react/status.coffee

```diff
@@ -0,0 +1,272 @@
+###
+# Copyright 2015 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community contributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+{div, span, br, strong, h1, h4, h5} = React.DOM
+el = React.createElement
+
+class @Status.Page extends React.Component

```

Comment:
this should be in `resources/assets/coffee/react/status/page.coffee` (or `main.coffee` just like two other pages)